src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -11,14 +11,14 @@ class Property
     /**
      * Reserved property keys for internal use which would be overwritten when importing
      *
-     * /image\d+/: Image URLs of type default.
-     * /thumbnail\d+/: Image URLs of type thumbnail.
-     * /ordernumber/: The products first exported ordernumber.
+     * - Image URLs of type default.
```

Comment:
:information_source: There is no benefit in repeating the patterns in the doc block, it just makes it easier for documentation to become outdated, so I removed it.